<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1164</width>
    <height>891</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Invoice Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="startWidget">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>366</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QPushButton" name="createDbButton">
              <property name="text">
               <string>Create Invoice Database</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="openDbButton">
              <property name="text">
               <string>Open Invoice Database</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="quitButton">
              <property name="text">
               <string>Quit</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>365</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="createWidget">
       <widget class="DbStatusForm" name="dbStatusWidget" native="true">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>620</y>
          <width>1021</width>
          <height>201</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>30</y>
          <width>170</width>
          <height>46</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>25</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Initial Data</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>410</y>
          <width>471</width>
          <height>181</height>
         </rect>
        </property>
        <property name="title">
         <string>Invoice CSS Templates</string>
        </property>
        <widget class="QWidget" name="verticalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>50</y>
           <width>321</width>
           <height>80</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="stylesheetsLayout"/>
        </widget>
        <widget class="QPushButton" name="addStylesheetButton">
         <property name="geometry">
          <rect>
           <x>387</x>
           <y>140</y>
           <width>31</width>
           <height>34</height>
          </rect>
         </property>
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>110</y>
          <width>350</width>
          <height>290</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Your Company Info</string>
           </property>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0" colspan="2">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Enter your company info here :</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Company Name</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="nameEdit"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Phone Number</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="phoneEdit"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Email</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="emailEdit"/>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Address</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLineEdit" name="addressLineEdit1"/>
            </item>
            <item row="5" column="1">
             <widget class="QLineEdit" name="addressLineEdit2"/>
            </item>
            <item row="6" column="1">
             <widget class="QLineEdit" name="addressLineEdit3"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="mainWidget">
       <widget class="QTableWidget" name="invoiceContentWidget">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>240</y>
          <width>671</width>
          <height>361</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>180</y>
          <width>88</width>
          <height>34</height>
         </rect>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
       <widget class="DbStatusForm" name="dbStatusWidget_2" native="true">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>670</y>
          <width>981</width>
          <height>171</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DbStatusForm</class>
   <extends>QWidget</extends>
   <header>DbStatusForm.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
